staging/src/k8s.io/client-go/examples/create-update-delete-service/README.md

```diff
@@ -0,0 +1,80 @@
+# Create, Update & Delete Service
+
+This example program demonstrates the fundamental operations for managing on
+[Service][1] resources, such as `Create`, `List`, `Update` and `Delete`.
+
+You can adopt the source code from this example to write programs that manage
+other types of resources through the Kubernetes API.
+
+## Running this example
+
+Make sure you have a Kubernetes cluster and `kubectl` is configured:
+
+    kubectl get nodes
+
+Compile this example on your workstation:
+
+```
+cd create-update-delete-service
+go build -o ./app
+```
+
+Now, run this application on your workstation with your local kubeconfig file:
+
+```
+./app
+# or specify a kubeconfig file with flag
+./app -kubeconfig=$HOME/.kube/config
+```
+
+Running this command will execute the following operations on your cluster:
+
+1. **Create service:** This will create a  Service. Verify with
+   `kubectl get services`.
+2. **Update Service:** This will update the Service resource created in
+   previous step by changing the type of service
+   `kubectl describe service demo`.
+3. **List Services:** This will retrieve Services in the `default`
+   namespace and print their names and type.
+4. **Delete Service:** This will delete the Service object. Verify with `kubectl get services`.
+
+Each step is separated by an interactive prompt. You must hit the
+<kbd>Return</kbd> key to proceed to the next step. You can use these prompts as
```

Comment:
I think Enter is more common
https://pc.net/helpcenter/answers/keyboard_return_key

https://en.wikipedia.org/wiki/Return_key redirects to https://en.wikipedia.org/wiki/Enter_key on Wiki